Makefile

```diff
@@ -30,6 +30,7 @@ TIMESCALEDB_BUILDER_URL?=$(TIMESCALEDB_IMAGE):builder-$(PGVERSION)
 TIMESCALEDB_RELEASE_URL?=$(TIMESCALEDB_IMAGE):$(TAG)-$(PGVERSION)
 TIMESCALEDB_LATEST_URL?=$(TIMESCALEDB_IMAGE):latest-$(PGVERSION)
 PG_PROMETHEUS?=0.2.2
+TIMESCALE_PROMETHEUS?=master
```

Comment:
Would be great if we could point to a tagged release here instead of master, even if that tag is a development version